### ISSUE TITLE
Questions about test results based on pre-trained model in README

### ISSUE THREAD

**Issue by HornGate** [Penn000/SpA-GAN_for_cloud_removal#1]
I use your pre-trained model, but I can't get the same results as that in README. Please help me. Thank you. The following images are my test results.
![1](https://user-images.githubusercontent.com/32919443/59337321-ee746f00-8d32-11e9-96ca-ee57b85e5804.png)
![2](https://user-images.githubusercontent.com/32919443/59337369-0815b680-8d33-11e9-9335-9eaf6d26dd3e.png)

**Comment by Penn000** [comment]
the pre-trained model was trained by normalizing the input image to [0, 1], while the old version code normalized the input to [-1, 1], the bug has been fixed now.

**Comment by pierowu** [comment]
> I use your pre-trained model, but I can't get the same results as that in README. Please help me. Thank you. The following images are my test results. ![1](https://user-images.githubusercontent.com/32919443/59337321-ee746f00-8d32-11e9-96ca-ee57b85e5804.png) ![2](https://user-images.githubusercontent.com/32919443/59337369-0815b680-8d33-11e9-9335-9eaf6d26dd3e.png)

Hi. Can you get the same quantitative results as the paper? I use the pre-model but can't get the corresponding results.
